03Constants.go

```diff
@@ -1,3 +1,4 @@
+//hi
```

Comment:
```suggestion
// Hi
```

Wow, a super big issue, thanks for submitting. 

but I think you need to add space after slices for commenting